webdriver/tests/support/fixtures_bidi.py

```diff
@@ -28,3 +28,39 @@ async def on_event(method, data):
 
         return future
     return wait_for_event
+
+
+@pytest.fixture
+async def context_id(send_blocking_command):
+    """Returns an `id` of an open context."""
+    response = await send_blocking_command("browsingContext.getTree", {})
+    if len(response['contexts']) < 1:
+        raise "No open contexts"
+    else:
+        return response['contexts'][0]['context']
```

Comment:
I don't see that `If parent is null, let child contexts be a list containing all top-level browsing contexts` will return the list in the correct order and that the first entry is always the current tab, especially when multiple windows are open. So the above should only be correct when there is a single window with one or more tabs open.

Reply:
for the test purposes we don't care (do we?) that much if the tab is a currently active one or not. We just need a tab to run tests on.